manim/mobject/coordinate_systems.py

```diff
@@ -256,28 +265,86 @@ def create_axis(self, range_terms, axis_config, length):
             axis.shift(-axis.number_to_point(0))
         return axis
 
-    def coords_to_point(self, *coords):
+    def coords_to_point(self, *coords: Sequence[float]) -> np.ndarray:
+        """Returns back a point with respect to the origin from the given coordinates.
+
+        Returns
+        -------
+        np.ndarray
+            A point formed by the coordinates
+        """
         origin = self.x_axis.number_to_point(0)
         result = np.array(origin)
         for axis, coord in zip(self.get_axes(), coords):
             result += axis.number_to_point(coord) - origin
         return result
 
-    def point_to_coords(self, point):
+    def point_to_coords(self, point: float) -> Tuple:
+        """Returns back the coordinates of a particular point.
+
+        Parameters
+        ----------
+        point
+            The point for finding the coordinates.
+
+        Returns
+        -------
+        Tuple
+            The coordinates of the points.
+        """
         return tuple([axis.point_to_number(point) for axis in self.get_axes()])
 
-    def get_axes(self):
+    def get_axes(self) -> VGroup:
+        """Gets the axes.
+
+        Returns
+        -------
+        :class:`~.VGroup`
+            Returns a pair of axes.
```

Comment:
`ThreeDAxes` has 3 axes, so `pair of axes` isn't appropriate.

Reply:
For this function it only returning 2

Reply:
It might look like its returning 2 since it is a method of `Axes` which has 2 axes, but that's not the case with `ThreeDAxes` (it has 3 axes). It would make sense to keep it as `A pair of axes` if `ThreeDAxes` had its own `get_axes` method, but since there isn't (I don't think its necessary), it doesn't look appropriate to me.